### ISSUE TITLE
sopra pool control

### ISSUE THREAD

**Issue by sg-sopraAG** [luconlua/sopra-cloud#65]
Status Licht:
- werden die Buttons S1 - S 4 sich nach einer Zuordnung der Farben auch in diese Farbe verfärben?
- Es wäre warscheinlich besser die Farben (rot,grün,blau,weiß) über zusätzliche Buttons fest zu legen und die S1-S4 als (Option frei programmierbar zu lassen) ist das machbar?
![sopra pool control-Anzeige-5](https://user-images.githubusercontent.com/67409030/88169830-e74daf00-cc1c-11ea-8ad2-99649544e0ac.png)

**Comment by wtuz** [comment]
Die Buttons sind momentan fixe Bilder. Hier wird momentan nichts eingefärbt.

Jedem Button (S1 - S4) ist eine frei konfigurierbare Farbe zugeordnet:
![image](https://user-images.githubusercontent.com/35223218/88185185-ee33ec00-cc33-11ea-8ac8-bd1c16a2c133.png)
Also Farbe 1 = S1
Farbe 2 = S2 usw.


Die Möglichkeit, wie wir es auf den internen Webseiten des Gerätes haben (siehe nachfolgendes Bild), den Button in der eingestellten Farbe anzuzeigen gibt es momentan in der Cloud nicht.
![image](https://user-images.githubusercontent.com/35223218/88184991-af9e3180-cc33-11ea-9001-5aee1fcf90fc.png)

Ggf. müsste so ein Feature noch in der Cloud implementiert werden

**Comment by sg-sopraAG** [comment]
Ja... so stelle ich mir das aber auch auf der cloud so vor und das Feature sollte dann so implementiert werden.
